### ISSUE TITLE
strings ending in ... arent detected

### ISSUE THREAD

**Issue by ImLunaHey** [edvardchen/eslint-plugin-i18next#138]
the following isnt being picked up. if i remove some of the periods it does. 

```tsx
if (isLoading) return <div>Loading...</div>;
```

**Comment by mlahlou-numendo** [comment]
i concur, with the following i respectively get :

```
export const Component = () => <p>TEST</p>
```
no error

```
export const Component = () => <p>TEST.</p>
```
error

```
export const Component = () => <p>TEST..</p>
```
no error

**Comment by sam6321** [comment]
I also ran into this issue and tried to debug it a bit. 

It's happening in `matchPatterns` when the literal string is checked.
This RegExp exclude that is part of the `options.words` array when `isValidLiteral` is being called: 
```
/(^|\.)[0-9!-/:-@[-`{-~]+$/
```

<img width="919" height="222" alt="Image" src="https://github.com/user-attachments/assets/d8524dba-da91-4623-80d0-5c252f17d3ac" />

The above pattern in the screenshot has `/(^|\.)` prepended to it by the `generateFullMatchRegExp` call.

This pattern will now trigger on any string that ends with multiple subsequent `.` characters, causing the string to be excluded and not checked.

**Comment by derpondus** [comment]
@edvardchen 
Any timeline on when this is released into the version on npm?
As of now, it isnt included in the version obtained by `npm install --save-dev eslint-plugin-i18next`

**Comment by edvardchen** [comment]
@derpondus I just released v6.1.5 which contains the fix. I forgot to run the release job previously

**Comment by derpondus** [comment]
No issue! Thanks for the fast response.
